packages/ui-drawer-layout/src/DrawerLayout/DrawerContent/index.js

```diff
@@ -84,16 +90,19 @@ class DrawerContent extends Component {
       trailing: true
     })
     this._resizeListener = addResizeListener(this._content, this._debounced)
+    this.props.makeStyles({ shouldTransition: false })
   }
 
   componentDidUpdate() {
+    /*
     this._timeouts.push(
       setTimeout(() => {
         this.setState({
           shouldTransition: true
         })
       })
-    )
+    )*/
```

Comment:
I think we should delete unused/commented out code, if we might need it, it is there in the git history.

Reply:
yea, this was left by accident